### DISCUSSION TITLE
Just a big Thank you

### DISCUSSION THREAD

**Discussion by modellbobby** [LMS-Community/slimserver#856] [General]
Dear Developers. 

I am an ex Roon Audirvana user and just installed LMS on my nuc and integrated Spotify and material skin. 

Man, this is just perfect !!! You totally created an amazing amazing !! Software. 

I want to thank you deeply for this. 

Kind regards
Modelbobby

**Comment by UHFRadio** [comment]
Ditto. Big thanks to Michael Herger for his longstanding support of this project. As a 15+ year owner-user of the Squeezebox line of products (many throughout the house) I have run LMS on Windows, Synology, and yes even on the Raspberry Pi !  Great to see LMS continuing to improve. Thanks again.
